external-crates/move/crates/move-ir-to-bytecode/src/compiler.rs

```diff
@@ -323,10 +325,18 @@ fn constant_name_as_constant_value_index(
     context.constant_index(name_constant)
 }
 
-/// Compile a module.
 pub fn compile_module<'a>(
     module: ModuleDefinition,
     dependencies: impl IntoIterator<Item = &'a CompiledModule>,
+) -> Result<(CompiledModule, SourceMap)> {
+    compile_module_with_version_opt(module, dependencies, None)
+}
+
+/// Compile a module.
+pub fn compile_module_with_version_opt<'a>(
+    module: ModuleDefinition,
+    dependencies: impl IntoIterator<Item = &'a CompiledModule>,
+    version: Option<u32>,
```

Comment:
Do we want to just stick this in `ModuleDefinition`? I know it will make this a bigger change, but it feels like the right thing long term...

Reply:
Had a look, and sadly this isn't possible :'(

`ModuleDefinition` is in `move-ir-types` so if we added this to that it would create a circular crate dependency + it would really be the wrong place for this function to live I feel.